R/init-package.R

```diff
@@ -18,6 +18,4 @@ initPackage <- function() {
   apiConfig$pkParametersFilePath <- filePathFor("OSPSuite.PKParameters.xml")
 
   rClr::clrCallStatic("OSPSuite.R.Api", "InitializeOnce", apiConfig$ref)
-  dimensionTask <- getNetTask("DimensionTask")

```

Comment:
@Yuri05 THis was creating some issues. And then I wondered. Do we actually need to load if from .NET 
isn't it enough to have it a constant as it is now?